cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -1408,3 +1408,18 @@ def _capacity_reservation(self, cr_target):
                 }
             )
         }
+
+
+class RootVolumeEncryptionConsistencyValidator(Validator):
+    """Verify consistency on the Encryption parameter of all the specified RootVolumes of the queues."""
+
+    def _validate(self, encryption_settings: list):
+        reference_queue_name, reference_root_volume_encryption = encryption_settings.pop(0)
+        for queue in encryption_settings:
+            queue_name, root_volume_encryption = queue
+            if reference_root_volume_encryption != root_volume_encryption:
+                self._add_failure(
+                    f"The Encryption parameter of the root volume of the queue {queue_name} is not consistent "
```

Comment:
I'd change 
```
Please be sure this is expected.
```
with:
```
and may cause a problem in case of Service control policies (SCPs) enforcing encryption. Please be sure this is expected.
```

and let's ask for a final confirmation to @andrea-curtoni